pkg/templates/pdbmaxunavailable/template.go

```diff
@@ -0,0 +1,53 @@
+package pdbmaxunavailable
+
+import (
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/pdbmaxunavailable/internal/params"
+	pdbV1 "k8s.io/api/policy/v1"
+)
+
+const (
+	templateKey           = "pdb-max-unavailable"
+	maxUnavailableZeroMsg = "MaxUnavailable is set to 0"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Pod Disruption Budget Configuration Checks",
+		Key:         templateKey,
+		Description: "Flag non-optimal PDB configurations",
```

Comment:
"Non-optimal" is too vague for what the check does, imho. How about this?

```suggestion
		Description: "Flag PodDisruptionBudgets whose maxUnavailable value prevents pod disruptions.",
```